.github/workflows/sync_notebooks.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches-ignore: [main]
     paths:
       - "examples/use_cases/**/*.ipynb"
-      - "examples/tutorials/00_api_basics.ipynb"
+      - "examples/tutorials/*.ipynb"
```

Comment:
The intention is to control all tutorial notebooks in `examples/tutorials/`, to which notebooks in `examples/use-cases/` may also be absorbed as well.